pasqal_cloud/job.py

```diff
@@ -7,6 +7,11 @@
 from pasqal_cloud.errors import JobCancellingError
 
 
+class ResultType(TypedDict):
```

Comment:
How does this behave if we add new attributes to result later? Will this break this version of the SDK?